src/main/resources/META-INF/resources/webjars/service/prov/prov.js

```diff
@@ -1755,19 +1755,26 @@ define(function () {
 			// Check the availability of this instance for these requirements
 			current.disableCreate($popup);
 			$.ajax({
-				dataType: 'json',
-				url: REST_PATH + 'service/prov/' + current.model.subscription + '/' + type + '-lookup/?' + queriesArray.join('&'),
-				type: 'GET',
-				success: function (suggest) {
-					current[popupType + 'SetUiPrice'](suggest);
-					if (suggest && (suggest.price || ($.isArray(suggest) && suggest.length))) {
-						// The resource is valid, enable the create
-						current.enableCreate($popup);
-					}
-				},
-				error: () => current.enableCreate($popup)
-			});
-		},
+                dataType: 'json',
+                url: REST_PATH + 'service/prov/' + current.model.subscription + '/' + type + '-lookup/?' + queriesArray.join('&'),
+                type: 'GET',
+                success: function (suggest) {
+                    current[popupType + 'SetUiPrice'](suggest);
+                    if (suggest && (suggest.price || ($.isArray(suggest) && suggest.length))) {
+                        // The resource is valid, enable the create
+                        current.enableCreate($popup);
+                    }
+                    if (suggest === undefined ||  suggest.price.edition === undefined){
```

Comment:
Move this condition inside the above "if"
Simplify this condition when include the previous "if"